apps/api-documenter/src/markdown/MarkdownEmitter.ts

```diff
@@ -191,8 +192,13 @@ export class MarkdownEmitter {
       case DocNodeKind.InlineTag: {
         break;
       }
+      case DocNodeKind.BlockTag: {
+        const tagNode: DocBlockTag = docNode as DocBlockTag;
+        console.warn('Unsupported block tag: ' + tagNode.tagName);
+        break;
+      }
       default:
-        throw new Error('Unsupported element kind: ' + docNode.kind);
+        console.warn('Unsupported element kind: ' + docNode.kind);
```

Comment:
@halfnibble This change was not safe in my opinion:  If API Documenter does not recognize some part of the TSDoc syntax tree, that's a bug.  We would want people to report an error, so we can determine a correct handling of that node.

Whereas if API Documenter silently ignored the unrecognized node, then it would be simply omitted from the output.  It could be years before we eventually realized that we forgot to implement it.  Note that it's okay to ignore an unsupported tag; here I'm referring to a generalized node in the tree.

This concern is particularly important since currently API Extractor / API Documenter are serving as the reference implementation of the TSDoc standard.

Reply:
I pushed a fix.